src/Ml.Cli.WebApp/ClientApp/src/Server/Dataset/Edit/Edit.container.js

```diff
@@ -41,13 +41,23 @@ const lockDataset = (fetch, setState) => async (state, idDataset) => {
     setState({...state, ...data, openLockModal: false, dataset: {...state.dataset, isLock:true }});
 };
 
-export const Edit = ({fetch, state, setState, lock}) => <div className="edit-dataset">
-    <ConfirmModal isOpen={state.openLockModal} onCancel={lock.onCancel} onSubmit={lock.onSubmit}/>
-    <Title title={state.dataset.name} subtitle="Edition du dataset" goTo="/datasets" goTitle="Datasets" />
-    <FileUpload fetch={fetch} state={state} setState={setState}/>
-    {state.files.filesSend.length === 0 ? null : <FileList fetch={fetch} state={state} setState={setState} />}
-    <Lock state={state} onLockDataset={lock.onLockDataset} />
-</div>
+export const Edit = ({fetch, state, setState, lock}) => {
+    const isDisabled = state.files.filesSend.length === 0;
```

Comment:
**suggestion** : use optionals or defensive coding

Reply:
Won't fix: the component is dedicated to the Dataset page, which is not the objective of the PR. Its unique modification is due to the globalization of ConfirmModal component